octavia-cli/octavia_cli/templates/connection.yaml.j2

```
@@ -0,0 +1,42 @@
+name: {{ resource_name }}
+source_id: {{ source_id }}
```

```suggestion
definition_type: connection
source_id: {{ source_id }}
```

It'd be useful to add this field so that a deserializer can know which type of resource is described in the file and not rely and file name only.